gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/validator/CseValidRequestValidator.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse_valid.app.validator;
+
+import com.farao_community.farao.cse_valid.api.resource.CseValidFileResource;
+import com.farao_community.farao.cse_valid.api.resource.CseValidRequest;
+import com.farao_community.farao.cse_valid.app.exception.CseValidRequestValidatorException;
+import org.springframework.stereotype.Component;
+
+import java.util.StringJoiner;
+
+/**
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
+
+@Component
+public class CseValidRequestValidator {
+
+    public void validateCseValidRequest(CseValidRequest cseValidRequest, Boolean isExportCornerActive) throws CseValidRequestValidatorException {
+
+        if (cseValidRequest == null) {
+            throw new CseValidRequestValidatorException("Request is null");
+        }
+
+        final boolean cgmFileExists = fileExists(cseValidRequest.getCgm());
+        final boolean glskFileExists = fileExists(cseValidRequest.getGlsk());
+        final boolean importCracFileExists = fileExists(cseValidRequest.getImportCrac());
+
+        if (isExportCornerActive == null || !isExportCornerActive) {
```

Comment:
Maybe replace with `! Boolean.TRUE.equals(isExportCornerActive)` to perform both operations at once ? I let you decide which solution you prefer.